exercises/Daniel_Murphy/ex02_oo_basics/src/shape.h

```diff
@@ -0,0 +1,103 @@
+#ifndef MAIN_H
+#define MAIN_H
```

Comment:
Nice header guard. I was just about to point out that the name needed updating to SHAPE_H. But I remembered that we've started using "#pragma once" instead so please can you change to that. This is a preprocessor directive that achieves the same thing ie prevent the header being imported multiple times in a particular compilation.